cli/test.sh

```diff
@@ -33,19 +33,23 @@ t=$(mktemp -d glometest.XXX)
 trap "rm -rf -- '${t?}'" EXIT
 
```

Comment:
Maybe for a follow-up CL I think it'd be worthwhile to actually validate the test vectors.

This worked for me:

```python
>>> open('PRIVATE-KEY', mode='wb').write(binascii.unhexlify('77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a'))
32
>>> open('PUBLIC-KEY', mode='wb').write(binascii.unhexlify('de9edb7d7b7dc1b4d35b61c2ece435373f8343c85b78674dadfc7e146f882b4f'))
32
```
```shell
$ echo -n "The quick brown fox" | ./glome tag --key PRIVATE-KEY --peer PUBLIC-KEY 
9c44389f462d35d0672faf73a5e118f8b9f5c340bbe8d340e2b947c205ea4fa3
```

Reply:
Opened #50 for it.